pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -3,13 +3,16 @@
 from pydatastructs.utils.misc_util import _check_type, _comp
 from concurrent.futures import ThreadPoolExecutor
 from math import log, floor
+from types import *
```

Comment:
Do not do `*`, import only those things which you want to use. Avoid namespace pollution.